sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -111,13 +111,16 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
       |  SELECT key FROM gen_tmp ORDER BY key ASC;
     """.stripMargin)
 
-  test("multiple generator in projection") {
+  test("multiple generators in projection") {
+    // TGF with non-TGF in project is allowed in Spark SQL, but not in Hive
+    sql("SELECT explode(map(key, key)) as (k1, k2), key, value FROM src").collect()
```

Comment:
If we don't have a test for this elsewhere, then this should probably be in its own `test` block and use `checkAnswer`.